tests/testthat/test_base_spcv.R

```diff
@@ -0,0 +1,41 @@
+
+test_that("Nested SpRepCV works without errors", {
+
+  data(bc.task.spatial, package = "mlr", envir = environment())
+
+  lrn.ksvm = makeLearner("classif.ksvm",
+                          predict.type = "prob",
+                          kernel = "rbfdot")
+
+  ps = makeParamSet(makeNumericParam("C", lower = 1, upper = 1),
+    makeNumericParam("sigma", lower = 1, upper = 1))
+
+  ctrl = makeTuneControlRandom(maxit = 1)
+  inner = makeResampleDesc("SpCV", iters = 2)
+
+  wrapper.ksvm = makeTuneWrapper(lrn.ksvm, resampling = inner, par.set = ps,
+    control = ctrl, show.info = FALSE, measures = list(auc))
+
+  outer = makeResampleDesc("SpRepCV", folds = 2, reps = 2)
+
+  library(parallelMap)
+  parallelStart(mode = "multicore", level = "mlr.tuneParams", cpus = 2)
+
+  resa.svm.spatial = mlr::resample(wrapper.ksvm, bc.task.spatial,
+    resampling = outer, show.info = TRUE, measures = list(auc))
+  parallelStop()
```

Comment:
See above -- parallelization and checks.

Reply:
Did so, see https://github.com/mlr-org/mlr/blob/e4c026700acc840c8d53f8e9cdcd894e49483610/tests/testthat/test_base_spcv.R#L39 and
https://github.com/mlr-org/mlr/blob/e4c026700acc840c8d53f8e9cdcd894e49483610/tests/testthat/test_base_spcv.R#L24